cli/ballerina-cli/src/main/resources/new_cmd_defaults/gitignore

```diff
@@ -1 +1,2 @@
 target
+**/Config.toml
```

Comment:
Will this include the Config.toml in the pacakge root? 
https://www.atlassian.com/git/tutorials/saving-changes/gitignore#git-ignore-patterns
The correct pattern seems to be
```suggestion
Config.toml
```

Reply:
Yes, I used `**/Config.toml` as it includes files in the package root as well as files within folders. But `Config.toml` also seems to do the same. So I will change it.